CHANGELOG.md

```diff
@@ -30,6 +31,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fixed a bug in the STM32 port that caused the final result to never be returned.
 - Fix bug when building a binary using a 64-bit integer on a 32-bit CPU.
 
+### Breaking changes
```

Comment:
I would try to follow the Keep a Changelog spec (https://keepachangelog.com/en/1.1.0/).
I'd rather add this information to UPDATING.md.
So I suggest doing this:
1. Let's add to Changed: "(Breaking) Changed pico applications offset, se also UPDATING.md" or something like that
2. Let's describe in few words how to update an existing project.